var/spack/repos/builtin/packages/exaca/package.py

```diff
@@ -20,6 +20,7 @@ class Exaca(CMakePackage):
 
     version("master", branch="master")
     version("1.0.0", sha256="48556233360a5e15e1fc20849e57dd60739c1991c7dfc7e6b2956af06688b96a")
+    version("1.1.0", sha256="10106fb1836964a19bc5bab3f374baa24188ba786c768e554442ab896b31ff24")
```

Comment:
The convention is to list newer numbered versions first so please move to between lines 21 and 22.

Reply:
Done - thanks